data-collection/deploy/module-securityhub.yaml

```diff
@@ -0,0 +1,452 @@
+                Resource:
+                  - !Sub "${DestinationBucketARN}"
+        - PolicyName: !Sub "${CFDataName}-detail-StateMachineExecution"
+          PolicyDocument:
+            Version: "2012-10-17"
+            Statement:
+              - Effect: "Allow"
+                Action: "states:StartExecution"
+                Resource: !Sub "arn:aws:states:${AWS::Region}:${AWS::AccountId}:stateMachine:${ResourcePrefix}${CFDataName}-detail-StateMachine"
+    Metadata:
+      cfn_nag:
+        rules_to_suppress:
+          - id: W28 # Resource found with an explicit name, this disallows updates that require replacement of this resource
+            reason: "Need explicit name to identify role actions"
+
+  IAMRoleAmazonEventBridgeInvokeFirehose:
+    UpdateReplacePolicy: "Delete"
+    Type: "AWS::IAM::Role"
+    DeletionPolicy: "Delete"
+    Properties:
+      Path: "/service-role/"
+      MaxSessionDuration: 3600
+      RoleName: !Sub "${ResourcePrefix}${CFDataName}-EBInvoke_Firehose"
+      AssumeRolePolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+        - Action: "sts:AssumeRole"
+          Effect: "Allow"
+          Principal:
+            Service: "events.amazonaws.com"
+      Metadata:
+        cfn_nag:
+          rules_to_suppress:
+            - id: W28 # Resource found with an explicit name, this disallows updates that require replacement of this resource
+              reason: "Need explicit name to identify role actions"
+
+    KinesisFirehoseRole:
+      Type: "AWS::IAM::Role"
+      Properties:
+        Path: "/"
+        MaxSessionDuration: 3600
+        RoleName: "${ResourcePrefix}${CFDataName}-KinesisFireHoseRole"
+        Description: "Allows Kinesis Firehose to transform and deliver data to your\
+          \ destinations using CloudWatch Logs, Lambda, and S3 on your behalf."
+        Policies:
+          - PolicyDocument:
+              Version: "2012-10-17"
+              Statement:
+              - Resource:
+                - !Sub "arn:${AWS::Partition}:s3:::${CodeBucket}"
+                - !Sub "arn:${AWS::Partition}:s3:::${CodeBucket}*"
+                - !Sub "arn:${AWS::Partition}:logs:${AWS::Region}:${AWS::AccountId}:log-group:%FIREHOSE_POLICY_TEMPLATE_PLACEHOLDER%:log-stream:*"
+                - !Sub "arn:${AWS::Partition}:logs:${AWS::Region}:${AWS::AccountId}:log-group:/aws/kinesisfirehose/PUT-S3-SVAnj:log-stream:*"
+                - !Sub "arn:aws:lambda:${AWS::Region}:${AWS::AccountId}:function:${LambdaFunction}"
+                Action:
+                - "s3:PutObject"
+                - "s3:GetObject"
+                - "s3:ListBucketMultipartUploads"
+                - "s3:AbortMultipartUpload"
+                - "lambda:InvokeFunction"
+                - "s3:ListBucket"
+                - "lambda:GetFunctionConfiguration"
+                - "s3:GetBucketLocation"
+                - "logs:PutLogEvents"
+                Effect: "Allow"
+                Sid: "VisualEditor0"
```

Comment:
VisualEditor0 ?

Also can we split to several statements for better readability?

Reply:
removed VisualEditor0 and fixed better readability